e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -2018,6 +2018,84 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
       });
     });
   });
+
+  it("should not pass through null values to filters in custom url click behavior (metabase#25203)", () => {
+    const DASHBOARD_NAME = "Click Behavior Custom URL Dashboard";
+    const questionDetails = {
+      name: "Orders",
+      query: {
+        "source-table": ORDERS_ID,
+        aggregation: [
+          ["sum", ["field", ORDERS.TOTAL, null]],
+          ["sum", ["field", ORDERS.DISCOUNT, null]],
+        ],
+        breakout: [["datetime-field", ["field-id", ORDERS.CREATED_AT], "year"]],
```

Comment:
IIRC, `datetime-field` is a pretty old format and isn't actively used nowadays. It works for backward-compatibility reasons, but we probably should use a more recent version of MBQL here:

```js
[ "field", ORDERS.CREATED_AT, { "temporal-unit": "year" } ]
```